src/SonarScanner.MSBuild.PreProcessor/Resources.resx

```diff
@@ -531,4 +531,7 @@ If you already have a compatible java version installed, please add either the p
   <data name="ERROR_SonarSourcesAndTestsNotSupported" xml:space="preserve">
     <value>The arguments 'sonar.sources' and 'sonar.tests' are not supported. Please remove them and invoke the scanner again.</value>
   </data>
+  <data name="WARN_SonarQubeNearEndOfSupport" xml:space="preserve">
```

Comment:
Maybe call this `WARN_SonarQube89NearEndOfSupport` to be specific? 
This reads like you are deprecating SonarQube completely :S

Reply:
What about a more generic `WARN_SonarQubeVersionNearEndOfSupport`?

Reply:
I would prefer to be specific, as this does not exactly say what we are deprecating.